e2e/stories/dataset/output-record.e2e.ts

```diff
@@ -3,9 +3,6 @@ import * as Joi from "joi";
 import { Dataset } from "../../../src/api/dataops/dataset";
 import { cleaning, DEFAULT_DATASET, dom, init } from "../../teardowns";
 
-// tslint:disable-next-line:no-var-requires
-const joiAssert = require("joi-assert");
-
```

Comment:
I'd like to remove joiAssert from out tests since it hasn't been developed for last 6 years...